coreos-4/policies/SA-System_and_Services_Acquisition/component.yaml

```diff
@@ -21,11 +21,33 @@
         organization-defined personnel or roles a system and services
         acquisition policy is outside the scope of
         CoreOS configuration.
+    - key: a.1
+      text: |
+        Organizational processes to develop, document, and disseminate to
+        organization-defined personnel or roles a system and services
+        acquisition policy is outside the scope of
+        CoreOS configuration.
+    - key: a.2
+      text: |
+        Organizational processes to develop, document, and disseminate to
+        organization-defined personnel or roles a system and services
+        acquisition policy is outside the scope of
+        CoreOS configuration.
     - key: b
       text: |
         Organizational review and updates to the current system and services
         acquisition policy and procedures at an organization-defined frequency
         is outside the scope of CoreOS configuration.
+    - key: b.1
+      text: |
+        Organizational review and updates to the current system and services
+        acquisition policy and procedures at an organization-defined frequency
+        is outside the scope of CoreOS configuration.
+    - key: b.2
+      text: |
+        Organizational review and updates to the current system and services
+        acquisition policy and procedures at an organization-defined frequency
+        is outside the scope of CoreOS configuration.
```

Comment:
Looking at the FedRAMP templates at https://www.fedramp.gov/assets/resources/templates/FedRAMP-SSP-High-Baseline-Template.docx, appears we only need a response at the SA-1 (a b) level.

Reply:
@redhatrises and I spoke live. As we move towards OSCAL, this level of granularity is needed. TBD on whether to call them SI-1(b)(1) or SI-1(b.1), but loose agreement it is more important to get this merged for incremental progress. Not a one-way door.

Reply:
We are going to evaluate this later as it will most likely be needed for OSCAL.